internal_transfer_with_agreed_amount/views/account_payment_views.xml

```diff
@@ -1,20 +1,27 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
-    <record id="view_account_payment_form_inherit" model="ir.ui.view">
+    <record id="view_account_payment_form" model="ir.ui.view">
         <field name="name">account.payment.form.inherit</field>
         <field name="model">account.payment</field>
-        <field name="inherit_id" ref="account.view_account_payment_form"/>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
         <field name="arch" type="xml">
             <xpath expr="//button[@name='post']" position="after">
-                <button name="%(action_internal_transfer_multicurrency)d" type="action"
-                    class="oe_highlight" string="Confirm with an agreed amount"
+                <button
+                    name="%(action_internal_transfer_multicurrency)d"
```

Comment:
Prefer the syntax:

`name="external_id"`

Instead of:

`name="%(external_id)d"`

As the latter is a placeholder that inserts the action's actual database ID, which is less safe.

Same below.

```suggestion
                    name="internal_transfer_with_agreed_amount.action_internal_transfer_multicurrency"
```

Note that the full external ID (i.e. including module) needs to be provided when using this form, because, since it's will now be present into the view, the external ID's module can't be guessed.

Reply:
Changes applied.